src/layouts/DefaultLayout.vue

```diff
@@ -1,21 +1,23 @@
 <script lang="ts" setup>
-import Footer from '../components/Footer.vue';
-import Navbar from '../components/Navbar.vue';
-
-interface Props {
-  flexSpacer?: boolean
-}
-
-const props = defineProps<Props>()
+import NavbarSide from '../components/navbars/NavbarSide.vue';
+import NavbarBottom from '../components/navbars/NavbarBottom.vue';
 
 </script>
 
 <template>
-  <div class="flex flex-col min-h-screen dark:bg-gray-950">
-    <Navbar type="fixed" />
-    <div class="h-14"/>
-    <slot />
-    <div v-if="props.flexSpacer ?? true" class="flex-1"/>
-    <Footer/>
+  <div class="flex flex-col min-h-screen bg-white text-stone-900 m-0 p-0">
+    <div class="flex flex-row m-0 p-0">
+      <div class="collapse lg:visible">
```

Comment:
What are the implications of using `visibility: collapse` over `display: none`?

Reply:
[Relevant docs](https://tailwindcss.com/docs/visibility)

I just used `collapse` as it was the most sensible option out of the visibility options (`visible`, `invisible` and `collapse`) and didn't even consider using `display: none`. Looking at it now, `display: none` might be more suitable as it allows for other components to change their size. I'll try switching over to this.